### PR TITLE
sorter: adjust unified sorter config

### PR DESCRIPTION
**Pull request by liuzix**
### What problem does this PR solve? 
- Lack of the ability to locally configure `sort-dir`.
- Lack of verification of Unified Sorter related parameters.

### What is changed and how it works?
- Added some verifications for Unified Sorter parameters.
- Allows `sort-dir` to be locally configured by passing command line argument `sort-dir` to `cdc server`. To maintain backward compatibility, when the local `sort-dir` is unset, a cdc process defaults to the `sort-dir` of the first changefeed it encounters (the orignally logic).

### Check List 

Tests 

 - Unit test
 - Integration test

Related changes

 - Need to cherry-pick to the release branch

### Release note

- adjust Unified Sorter parameter settings.

### CODE REVIEW

File: cmd/server.go [hunk 1]
```diff
@@ -78,6 +78,7 @@ func initServerCmd(cmd *cobra.Command) {
 	cmd.Flags().IntVar(&serverConfig.Sorter.MaxMemoryPressure, "sorter-max-memory-percentage", defaultServerConfig.Sorter.MaxMemoryPressure, "system memory usage threshold for forcing in-disk sort")
 	// We use 8GB as a safe default before we support local configuration file.
 	cmd.Flags().Uint64Var(&serverConfig.Sorter.MaxMemoryConsumption, "sorter-max-memory-consumption", defaultServerConfig.Sorter.MaxMemoryConsumption, "maximum memory consumption of in-memory sort")
+	cmd.Flags().StringVar(&serverConfig.Sorter.SortDir, "sort-dir", defaultServerConfig.Sorter.SortDir, "sorter's temporary file directory")
```
Comment (amyangfei) [inline cmd/server.go:81]: Should we add one more command line parameter, is it enough to keep it in server config file?

**Review by leoppro — APPROVED** [review]

**Pull request merged**